itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/queryhistory/schema/ExampleQueryHistoryRecord.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.apache.hadoop.hive.ql.queryhistory.schema;
+
+import com.google.common.collect.Sets;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+public class ExampleQueryHistoryRecord extends QueryHistoryRecord {
```

Comment:
usually we call it`Dummy`: DummyQueryHistoryRecord

Reply:
agreed, but **Dummy** is an empty thing, here **Example** reflects that I intentionally filled all the fields in this record, does it make sense? but as we're not having a DummyQueryHistoryRecord yet, I'm ready to rename

Reply:
dummy doesn't mean empty it's a kinda stub (see DummyPartition)